lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.cpp

```diff
@@ -962,12 +952,11 @@ void ProxyShape::variantSelectionListener(SdfNotice::LayersDidChange const& noti
 }
 
 //----------------------------------------------------------------------------------------------------------------------
-void ProxyShape::reloadStage(MPlug& plug)
+void ProxyShape::loadStage()
```

Comment:
@dbaz I still like loadStage because: If the file name changes, its reading from a whole new source and USD treats it like a different stage in the cache. Part of the motivation for changing it was that it was getting called repeatedly (for other reasons than the file name change) and each time it was adding a new stage to the stage cache because they each had a unique anonymous session layer. Naming it loadStage() discourages using it simply to refresh the data on the same stage.

Reply:
Yep, sure. Thanks for the explanation.